nifi-nar-bundles/nifi-azure-bundle/nifi-azure-services-api/src/main/java/org/apache/nifi/services/azure/storage/ADLSCredentialsDetails.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.services.azure.storage;
+
+import com.azure.core.credential.AccessToken;
+
+public class ADLSCredentialsDetails {
+    private final String accountName;
+
+    private final String accountKey;
+    private final String sasToken;
+    private final String endpointSuffix;
+
+    private final AccessToken accessToken;
+
+    private final boolean useManagedIdentity;
+
+    public ADLSCredentialsDetails(
+        String accountName,
+        String accountKey,
+        String sasToken,
+        String endpointSuffix,
+        AccessToken accessToken,
+        boolean useManagedIdentity
+    ) {
+        this.accountName = accountName;
+        this.accountKey = accountKey;
+        this.sasToken = sasToken;
+        this.accessToken = accessToken;
+        this.endpointSuffix = endpointSuffix;
+        this.useManagedIdentity = useManagedIdentity;
```

Comment:
Could you please use the same order of the fields above (also adjust constructor parameters and assignments) as in the builder / getters (those looks fine)?